app/src/main/kotlin/com/wire/android/ui/authentication/devices/remove/RemoveDeviceScreen.kt

```diff
@@ -88,23 +96,27 @@ fun RemoveDeviceScreen() {
         onErrorDialogDismiss = viewModel::clearDeleteClientError,
         onBackButtonClicked = clearSessionViewModel::onBackButtonClicked,
         onCancelLoginClicked = clearSessionViewModel::onCancelLoginClicked,
-        onProceedLoginClicked = clearSessionViewModel::onProceedLoginClicked
+        onProceedLoginClicked = clearSessionViewModel::onProceedLoginClicked,
+        navigationIconType = NavigationIconType.Close
```

Comment:
Why was this value added from here? 🤔

Reply:
I did a refactor on the component, so some parts were maintained, and others modified...so this basically give us the ability to reuse the same screen and parametrize the navigation stuff. I.e.: on removing when logging in which changes the navigation icon (`X`) and navigation itself. While on removing from settings, we have the arrow icon (`<-`) and the navigation is back.